forui/test/src/widgets/line_calendar_golden_test.dart

```diff
@@ -0,0 +1,37 @@
+@Tags(['golden'])
+library;
+
+import 'package:flutter/widgets.dart';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:sugar/sugar.dart';
+
+import 'package:forui/src/widgets/line_calendar.dart';
+import '../test_scaffold.dart';
+
+void main() {
+  group('FLineCalendar', () {
+    for (final (name, theme, _) in TestScaffold.themes) {
+      for (final (lineCalendar, value) in [('default', ValueNotifier(LocalDate.now()))]) {
```

Comment:
I'm surprised this test doesn't fail since it's using `LocalDate.now()` which will change each day...